nipype/interfaces/fsl/tests/test_auto_ApplyXfm.py

```diff
@@ -1,9 +1,9 @@
 # AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
 from __future__ import unicode_literals
-from ..preprocess import ApplyXfm
+from ..preprocess import ApplyXFM
```

Comment:
i think this PR has taken care of it. the issue is with OSX and case-insensitive installations, which gave rise to the same named file with different content. the older interface which was deprecated in 0.12.1 has now been removed, so we should be ok.